client/my-sites/plugins/plugin-details-CTA/index.jsx

```diff
@@ -526,11 +531,11 @@ function ManageSitesButton( { plugin, installedOnSitesQuantity } ) {
 				</div>
 			) }
 			<Button
-				className="plugin-details-cta__manage-button"
+				className={ className }
```

Comment:
This is unexpected to me, I feel the convention when a component accepts a `className` prop is to merge the class names like `clsx( 'plugin-details-cta__manage-button', className )`